apps/storybook.namekit.io/stories/Namekit/Tooltip.stories.tsx

```diff
@@ -39,60 +26,88 @@ type Story = StoryObj<typeof Tooltip>;
 
 export const TopPlacement: Story = {
   args: {
-    trigger: <>Tooltip Trigger</>,
-    children: <>Tooltip content</>,
-    placement: TooltipPlacement.TOP,
+    placement: TooltipPlacement.Top,
+  },
+  render: (args) => {
+    const trigger = <span>Tooltip Trigger</span>;
+    return (
+      <Tooltip {...args} trigger={trigger}>
+        <span>Tooltip content</span>
+      </Tooltip>
+    );
   },
 };
 
 export const BottomPlacement: Story = {
   args: {
-    trigger: <>Tooltip Trigger</>,
-    children: <>Tooltip content</>,
-    placement: TooltipPlacement.BOTTOM,
+    placement: TooltipPlacement.Bottom,
+  },
+  render: (args) => {
+    const trigger = <span>Tooltip Trigger</span>;
+    return (
+      <Tooltip {...args} trigger={trigger}>
+        <span>Tooltip content</span>
+      </Tooltip>
+    );
   },
 };
 
 export const LeftPlacement: Story = {
   args: {
-    trigger: <>Tooltip Trigger</>,
-    children: <>Tooltip content</>,
-    placement: TooltipPlacement.LEFT,
+    placement: TooltipPlacement.Left,
+  },
+  render: (args) => {
+    const trigger = <span>Tooltip Trigger</span>;
+    return (
+      <Tooltip {...args} trigger={trigger}>
+        <span>Tooltip content</span>
+      </Tooltip>
+    );
   },
 };
 
 export const RightPlacement: Story = {
   args: {
-    trigger: <>Tooltip Trigger</>,
-    children: <>Tooltip content</>,
-    placement: TooltipPlacement.RIGHT,
+    placement: TooltipPlacement.Right,
+  },
+  render: (args) => {
+    const trigger = <span>Tooltip Trigger</span>;
+    return (
+      <Tooltip {...args} trigger={trigger}>
+        <span>Tooltip content</span>
+      </Tooltip>
+    );
   },
 };
 
 export const DefaultMaxTooltipWidth: Story = {
-  args: {
-    trigger: <>Tooltip Trigger</>,
-    children: (
-      <>
-        This is the default max tooltip width. You can customize the max width
-        if desired. See the "Custom Max Tooltip Width" story for an example of
-        setting a custom max width.
-      </>
-    ),
-    placement: TooltipPlacement.BOTTOM,
+  render: (args) => {
+    const trigger = <span>Tooltip Trigger</span>;
```

Comment:
Maybe moving this to a DEFAULT constant is a good idea?

Reply:
@FrancoAguzzi I thought about that too, but I'm happy with a little bit of repetition here because it makes the code so nice and simple.